### PR TITLE
Delete (Cluster)Role(Bindings) as a final cleanup step.

### PR DESCRIPTION
**Pull request by Cynocracy**
This allows us to take advantage of the permissions granted to cluster admins
when performing cleanup. If the approach in #109
is approved, this will become necessary. Additionally, it makes sense to remove roles as the final
step to allow human Operators to modify any resources they may have permissions on as a result of the Knative installation (that is, we should not remove any access until we are 'almost done' cleaning up).

/lint

## Proposed Changes

*Delete roles and rolebindings as a final cleanup step.

## Release Note

```release-note
Delete roles and rolebindings as a final cleanup step.
```

### CODE REVIEW

File: config/role.yaml [hunk 1]
```diff
@@ -26,9 +26,6 @@ rules:
   - apps
   resources:
   - deployments
-  - daemonsets
```
Comment (Cynocracy) [inline config/role.yaml]: whoops, this shouldn't be here, one moment..

**Comment by Cynocracy** [comment]
Whoops, looks like I need to update manifestival, one moment.

**Comment by Cynocracy** [comment]
Blocked on https://github.com/knative/eventing-operator/pull/142

**Comment by k4leung4** [comment]
is this still WIP?

**Comment by Cynocracy** [comment]
/assign @matzew

See also https://github.com/knative/serving-operator/pulls?q=is%3Apr+is%3Aclosed

**Comment by Cynocracy** [comment]
Oh, sorry kleung4, missed your comment, no it is no longer WIP.

**Comment by Cynocracy** [comment]
@googlebot I consent

**Pull request merged**